target/linux/ramips/dts/mt7620a_domywifi.dtsi

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		led-boot = &system_amber;
+		led-failsafe = &system_amber;
+		led-running = &system_green;
+		led-upgrade = &system_amber;
+		label-mac-device = &wmac;
```

Comment:
This does not work (label-mac-device only works for mtd-mac-address).
Either use &ethernet here if it's the same address, or use label_mac in 02_network otherwise.